content/articles/how-to-make-form-using-react/index.md

```diff
@@ -323,7 +323,114 @@ Here is the link to find the full code of this registration form
+
+![Real Time Database](./realtimedatabase.png)
+
+Then click on the "Create Database" button. The Set up database dialog box will appear 
+
+![Settings](./Settings.png)
+
+We will go to next step by leaving the default location. Since we are in developing mode, we will select "Start in test mode" and enable it.
+
+Now our place is ready where real time data is stored.
+image of real time data 
+
+We need to create linkage between our react project and real time data base. 
+ 
+- For that we will go to project overview from the left panel and then click on Web button.
+
+![Button](./Button.png)
+
+- We have to Register our app and give nickname to it. We are giving "Registration Form" or we can choose any name of our choice. Since we are not hosting we will not select the option "set up Firebase hosting for this app" and then click on Register App. 
+
+![Register App](./Registerapp.png)
+
+- We go to our react project there we will connect our real time database. For that we need to type this command into the terminal or we can copy paste it too.
+   
+`npm install firebase`
+
+- To add the package into your react app we have to install firebase package over there. Now using terminal or command prompt go to the directory where your project is and run this command. 
+ 
+![package](./package.png)
+
+Create a new file with the name "firebase.js" in the Src folder where we will copy the code. 
+
+we have initialize the fire base. Now we will import data base in our code. 
+- For importing the data write the following statement
+
+`import { getDatabase } from "firebase/database"`
+
+- In order to use this data base in our registration form component we will use  export statement. Add the following statements in the code.
+`export const database = getDatabase(app);`
+
+In the registration form component import the following files
+
+`import {database} from '../firebase'
+import {ref,push,child,update} from "firebase/database";`
+
+- To store the input entered by the user we need to add the following code into the handle Submit function we made earlier 
+
+```java script
+let obj = {
+            firstName : firstName,
+            lastName:lastName,
+            email:email,
+            password:password,
+            confirmPassword:confirmPassword,
+        }       
+        const newPostKey = push(child(ref(database), 'posts')).key;
+        const updates = {};
+        updates['/' + newPostKey] = obj
+        return update(ref(database), updates);
+```
+
+Npw we have completed our steps successfully. 
+
+****Now we will see how it work****
+
+We will fill the details in the form then click on register .
+
+![Details](./Details.png)
+
+When we go to your firebase base account and check your Realtime database the your input is available there as shown in below pic.
+
+![Datainfirebase](./Datainfirebase.png)
+
+Congratulation you have successfully learn how to make registration form and connect it to firebase to store real time data.
```

Comment:
There's supposed to be a conclusion section, summarizing what you have done in the tutorial.